webui/ocw/migrations/0004_auto_20190905_1552.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 2.1.7 on 2019-09-05 15:52
```

Comment:
do we really need to keep auto generated files in git ? can we just regenerate them when needed ?

Reply:
In general I would agree. But here it is different, as we run upgrades of already existing databases, which correlate to the current code update. So I think it doesn't harm to keep them.